src/ralph/models/xapi/lms/objects.py

```diff
@@ -0,0 +1,60 @@
+"""LMS xAPI events object fields definitions."""
+
+from typing import Optional
+
+try:
+    from typing import Literal  # pylint: disable = ungrouped-imports
+except ImportError:
+    from typing_extensions import Literal
+
+from pydantic import Field
+
+from ..concepts.activity_types.acrossx_profile import WebpageActivity
+from ..concepts.activity_types.activity_streams_vocabulary import FileActivity
+from ..concepts.constants.acrossx_profile import ACTIVITY_EXTENSIONS_TYPE
+from ..config import BaseExtensionModelWithConfig
+
+
+# Page
+class LMSPageObjectExtensions(BaseExtensionModelWithConfig):
+    """Pydantic model for LMS page `object`.`extensions` property.
+
+    Attributes:
+        type (str): Consists of the characterisation of the page. Can be either
+            `course`, `course_list`, `user_space` value.
+    """
+
+    type: Optional[Literal["course", "course_list", "user_space"]] = Field(
+        alias=ACTIVITY_EXTENSIONS_TYPE
+    )
+
+
+class LMSPageObject(WebpageActivity):
+    """Pydantic model for LMS page `object` property.
+
+    Attributes:
+        extensions (dict): see LMSPageObjectExtensions.
+    """
+
+    extensions: Optional[LMSPageObjectExtensions]
+
+
+# File
+class LMSFileObjectExtensions(BaseExtensionModelWithConfig):
+    """Pydantic model for LMS file `object`.`extensions` property.
+
+    Attributes:
+        type (str): Consists of the characterisation of the MIME type of the file
+    """
+
+    type: str = Field(alias=ACTIVITY_EXTENSIONS_TYPE)
+
+
+class LMSFileObject(FileActivity):
```

Comment:
```suggestion
class LMSFileObjectDefinition(FileActivity):
```
Isn't it the `object`.`definition` field that contains extensions?